### DISCUSSION TITLE
HTTP Error 500 on LOTW Upload

### DISCUSSION THREAD

**Discussion by KE8AKW** [magicbug/Cloudlog#2006] [General]
Hi,
I am running Cloudlog locally on a windows machine via wampserver and I get "This page isn’t working" HTTP ERROR 500 if I try to manually sync LOTW or if I run it via cronjob. Checking the server logs I get this:  Severity: error --> Exception: SSL certificate problem: self signed certificate in certificate chain C:\wamp64\www\Cloudlog\application\controllers\Lotw.php 330
How do I fix this?

**Comment by magicbug** [comment]
Quite likely an OpenSSL issue but I've never tried to run this under wamp so not sure how to best suggest a fix

**Comment by AndreasK79** [comment]
Take a look here: https://stackoverflow.com/a/23585500

**Comment by KE8AKW** [comment]
Figured it out. Used the link that Andreas provided still dident solve my issue by adding a path for cacert.pem in the php.ini folder, what actually fixed it was adding a path for cacert.pem in a file called phpForApache.ini. That was a big head scratcher for a while, thanks!

**Reply by AndreasK79** [reply]
Good to hear that you figured it out :)
